docs/user_guide.rst

```diff
@@ -222,6 +222,20 @@ a ``.saucelabs`` configuration file in the working directory or your home
 directory, or by setting the ``SAUCELABS_USERNAME`` and ``SAUCELABS_API_KEY``
 environment variables.
 
+Alternatively, when using `Jenkins CI`_ declarative pipelines,
+credentials can be set as environment variables as follows:
+
+.. code-block:: groovy
+
+  environment {
+    SAUCELABS = credentials('SAUCELABS')
+  }
+
+For more information see Jenkins `docs`_.
+
+.. _Jenkins CI: <https://jenkins.io/>
+.. _docs: <https://jenkins.io/doc/pipeline/tour/environment/>
```

Comment:
These aren't working for me in GitHub's view. I think it might be the `<>` around the URLs? Also, let's put these at the end of the file, and could you make the link text/reference for the environment docs more descriptive? Something like:

```rst
For more information, see `using environment in Jenkins pipelines`_.

.. _using environment variables in Jenkins pipelines: https://jenkins.io/doc/pipeline/tour/environment/
```